azext_iot/_help.py

```diff
@@ -1070,6 +1070,9 @@
 
                   Edge deployments can be created with user defined metrics for on demand evaluation.
                   User metrics are json and in the form of {"queries":{...}} or {"metrics":{"queries":{...}}}.
+
+                  Deployments are classified as layered only if the content has $edgeAgent and within the $edgeAgent, properties.desired.x is defined,
```

Comment:
nits around increasing specificity a bit

"...and within the $edgeAgent **object**,"

". If properties.desired: {} is defined, the deployment will be treated as full (non-layered)."

We should also include a statement around $edgeAgent and $edgeHub must both be in either layered or full deployment format.

Reply:
It only needs the edgeAgent to be considered a layered deployment:
![image](https://user-images.githubusercontent.com/73560279/227373969-60705f4a-21d4-4707-a7a6-778ab0b3ad99.png)
![image](https://user-images.githubusercontent.com/73560279/227374028-912fcd4f-7715-40cc-a818-adc25a87a8dc.png)

Reply:
Resolved.